### PR TITLE
prepare changelog for release v0.82.0

### LINKED ISSUE #1994: Release v0.82.0
[Release v0.82.0](https://github.com/open-telemetry/opentelemetry-collector/issues/8139)

### PR DESCRIPTION
**Pull request by frzifus**
Closes  #1994

### CODE REVIEW

**Review by swiatekm-sumo — COMMENTED** [review]
Looking at the release checklist, you still need to:
4. Change the compatibility matrix in the [readme](./README.md) file, using the OpenTelemetry Operator version to be released and the current latest Kubernetes version as the latest supported version. Remove the oldest entry.
10. Update release schedule table, by moving the current release manager to the end of the table with updated release version.

Technically that last one should happen after the release is successful, but last time Tyler did it immediately, and I don't see any reason to make this change separately.

**Review by jaronoff97 — APPROVED** [review]

**Comment by frzifus** [comment]
mhpf... seems the integration tests are failing :(

File: versions.txt [hunk 1]
```diff
@@ -2,32 +2,32 @@
...
 
 # Represents the current release of Java instrumentation.
 # Should match autoinstrumentation/java/version.txt
-autoinstrumentation-java=1.26.0
+autoinstrumentation-java=1.28.0
 
 # Represents the current release of NodeJS instrumentation.
 # Should match value in autoinstrumentation/nodejs/package.json
-autoinstrumentation-nodejs=0.40.0
+autoinstrumentation-nodejs=0.41.0
```
Comment (swiatekm-sumo) [inline versions.txt:22]: This is why e2e tests are failing.

```suggestion
autoinstrumentation-nodejs=0.41.1
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,38 @@ Changes by Version
...
+- `target allocator`: Make the Target Allocator default scrape interval for Prometheus CRs configurable (#1925)
+  Note that this only works for Prometheus CRs, raw Prometheus configuration from the receiver uses its own settings.
+- `operator`: Set securityContext on injected initContainer based on existing containers. (#1084, #1058)
+- `Documentation`: Update OTel Operator and Target Allocator readmes. (#1952)
+
+### 🧰 Bug fixes 🧰
+
+- `operator`: Fix port name matching between ingress/route and service. All ports are truncated to 15 characters. If the port name is longer it is changed to port-%d pattern. (#1954)
+- `operator`: Fix for issue #1893 (#1905)
+
```
Comment (TylerHelmuth) [inline CHANGELOG.md:37]: Please manually add the `Components` section here, similar to the one in 0.81.0.

Comment (frzifus) [reply CHANGELOG.md:37]: done :+1:

**Review by pavolloffay — APPROVED** [review]

**Pull request merged**